controller/state/migrations/000003_initialize_schema.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+DROP TABLE IF EXISTS drainedWorkers;
```

Comment:
To be consistent with other table names, use: `drained_workers`

... although I just realized we have `finalizedData`.